test_discovery.py

```diff
@@ -263,6 +268,32 @@ def run_all_tests_in_directory(directory, custom_directory, tests, shutdown):
     return success
 
 
+def expand_wildcard_tests(directory, tests):
+    """
+    Expands any wildcard filenames in the list of tests
+    
+    >>> expand_wildcard_tests('examples/fruits', ['test*'])
+    ['tests.txt', 'tests_negative.txt']
+    
+    >>> expand_wildcard_tests('skip_this_directory', ['no_wildcards'])
+    ['no_wildcards']
+    """
+    if any('*' in t for t in tests):
+        wildcard_tests = [t for t in tests if '*' in t]
+        expanded_filenames = [ \
+            glob.glob(join_path(directory, add_extension_if_missing(t))) \
+            for t in wildcard_tests \
+        ]
+        flattened_filenames = [i for s in expanded_filenames for i in s]
```

Comment:
variable names in place of `i` or `s`.

Reply:
Again, this is simply to flatten the nested list. I don’t know what value variable names provide here.